### PR TITLE
Conform error page to new design

### PR DESCRIPTION
**Pull request by maiste**
This PR intends to conform the error page to the new design.

Here is what it looks like:
image

### CODE REVIEW

File: web-ui/view/client_error.ml [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
+                  [
+                    "flex justify-center flex-col items-center fixed left-0 \
+                     top-0 w-full h-full px-10";
+                  ];
+              ]
+            [
+              h1 ~a:[ a_class [ "text-stroke" ] ] [ txt (Fmt.str "%d" code) ];
+              div
+                ~a:[ a_class [ "text-2xl font-medium my-6" ] ]
+                [ txt (Fmt.str "Opps! %s" reason) ];
```
Comment (benmandrew) [inline web-ui/view/client_error.ml:67]: Spelling: 'Oops!'

**Review by benmandrew — APPROVED** [review]
Looks good to me apart from one spelling error.

File: web-ui/view/client_error.ml (hunk 1 above)
Comment (maiste) [reply web-ui/view/client_error.ml]: Copy & past of the mock-up haha: updated.

File: web-ui/view/client_error.ml [hunk 2]
```diff
@@ -1,12 +1,89 @@
...
+            ~a:
+              [
+                a_class
+                  [
+                    "flex justify-center flex-col items-center fixed left-0 \
+                     top-0 w-full h-full px-10";
+                  ];
+              ]
+            [
+              h1 ~a:[ a_class [ "text-stroke" ] ] [ txt (Fmt.str "%d" code) ];
```
Comment (novemberkilo) [inline web-ui/view/client_error.ml:64]: Oh cool so this just renders whatever code is passed in. Nice. Did you get a chance to check that this is all ok for 500s as well?

**Review by novemberkilo — APPROVED** [review]
Looks good thanks for covering this @maiste 

Figuring this is the case but could you please confirm that this is what is used when `Dream.error` is triggered? And that there are no more instances of the "old" error page?

Please :ship: when ready

File: web-ui/view/client_error.ml (hunk 2 above)
Comment (maiste) [reply web-ui/view/client_error.ml:64]: Couldn't reproduce one on my side... That's why I wanted to merge it: to make sure it apply.
And yes, this is the error handler given to `Dream` :)

**Pull request merged**